### DISCUSSION TITLE
Combine phi1 and phi2 ODF-sections plot

### DISCUSSION THREAD

**Discussion by sebastbg** [mtex-toolbox/mtex#1081] [Ask Anything]
Hey

Is there a nice way to combine ODF-sections plot for phi1 and phi2?
I want to plot the phi2 = 45 section and phi1 = 60, 75 and 90 sections in a [1 4] layout with annotations and legend. 
The only solution I have found is to save the generated figure and combined it afterwards.

Kind Regards 
Sebastian

**Comment by ralfHielscher** [comment]
No, there is not simple way to do this.

Ralf.
